src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -53,23 +54,43 @@ def test_invalidfiles(invalid_file):
 
 def test_dHdl_invalidfiles(invalid_file):
     """Test if we catch possible parsing errors in invalid files"""
-    assert extract_dHdl(invalid_file, T=300) is None
+    assert extract_dHdl(invalid_file, T=298.0) is None
 
 
 def test_dHdl_time_reading(single_dHdl, first_time=22.0, last_time=1020.0):
     """Test if time information is read correctly when extracting dHdl"""
-    dHdl = extract_dHdl(single_dHdl, T=300)
+    dHdl = extract_dHdl(single_dHdl, T=298.0)
     assert_allclose(dHdl.index.values[0][0], first_time)
     assert_allclose(dHdl.index.values[-1][0], last_time)
 
 
 def test_u_nk_time_reading(single_u_nk, first_time=22.0, last_time=1020.0):
     """Test if time information is read correctly when extracting u_nk"""
-    u_nk = extract_u_nk(single_u_nk, T=300)
+    u_nk = extract_u_nk(single_u_nk, T=298.0)
     assert_allclose(u_nk.index.values[0][0], first_time)
     assert_allclose(u_nk.index.values[-1][0], last_time)
 
 
+def test_wrong_T_should_raise_warning_in_extract_dHdl(single_dHdl, T=300.0):
+    """
+    Test if calling extract_dHdl with differnt T from what's
+    read from the AMBER file gives a warning
+    """
+    with pytest.raises(ValueError) as exc_info:
```

Comment:
with pytest.raises(ValueError, match="is different from the temperature passed as parameter"):

Reply:
Thanks, changed that, will be in next commit!